### ISSUE TITLE
Error: Collector Creation in Security Posture Management API service:  `us` region not supported

### ISSUE THREAD

**Issue by srmamit** [IBM-Cloud/terraform-provider-ibm#3630]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform IBM Provider Version
Terraform CLI version: 1.0.0
Terraform IBM Provider Version: 1.39.0

### Affected Resource(s)

* ibm_scc_posture_collector

### Terraform Configuration Files

```
resource "ibm_scc_posture_collector" "collectors" {
  description = "tf sample collector"
  is_public = true
  managed_by = "ibm"
  name = "tf-IBM-collector-sample-resource"
  is_ubi_image = true
}
```
### Debug Output

```
│ Error: [ERROR] Error occurred while configuring Security Posture Management API service:  `us` region not supported
│ 
│   with ibm_scc_posture_collector.collectors,
│   on main.tf line 6, in resource "ibm_scc_posture_collector" "collectors":
│    6: resource "ibm_scc_posture_collector" "collectors" {
```

### Expected Behavior

It would be expected to create a collector in US region.

### Actual Behavior

Errpr in security posture management API service, us region was not found.

### Steps to Reproduce

1. `terraform apply`

### References

1. https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/scc_posture_collector
2. https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/data-sources/scc_account_locations

**Comment by hkantare** [comment]
Fixed in https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.39.2

**Comment by srmamit** [comment]
resolved in v1.39.2
